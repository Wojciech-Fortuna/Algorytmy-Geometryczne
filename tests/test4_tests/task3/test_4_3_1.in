-0.034729840063279685 0.019299425704806472 0.030693547033494514 -0.03327410370695825
-0.02962903361166678 -0.03563317233440923 0.03379838574317193 0.028735700214610402
0.01117741800123645 0.031431778645982955 0.012064514775429996 -0.032937093903036674
0.02226612767865581 0.028061680606767267 0.022709676065752582 -0.03394812331480138
-0.024306452966505486 -0.015412584099115098 -0.017431452966505487 -0.007661358608919019