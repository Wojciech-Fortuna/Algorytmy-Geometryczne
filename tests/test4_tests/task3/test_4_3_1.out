-0.0016743790516658117 -0.007263586832941072 1 2
0.011526074740254208 0.006132801296422439 2 3
0.011860591253227505 -0.018140143229495424 1 3
0.022344451130778698 0.01711175484919488 2 4
0.022658670631229855 -0.026817363547212168 1 4